### PR TITLE
Maintenance in preparation for v1.5

### PR DESCRIPTION
**Pull request by arunkannawadi**
This is the third and the last PR (from me) towards releasing v1.5 of the package. Fixes several bugs ( #313 , #366 ) and includes various enhancements and deprecates Tor methods.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - given-names: Arun
+    family-names: Kannawadi
+    email: arunkannawadi@astro.princeton.edu
+    affiliation: >-
+      Department of Astrophysical Sciences, Princeton
+      University, 4 Ivy Lane, Princeton NJ 08544 USA
+    orcid: 'https://orcid.org/0000-0001-8783-6529'
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.1234
```
Comment (arunkannawadi) [inline CITATION.cff:36]: I have a dummy value here for now. @silvavn where should we expect the DOI to show up once we release this?

**Comment by arunkannawadi** [comment]
I'm not sure why, but this branch has become protected and I am unable to force-push commits anymore

**Review by silvavn — APPROVED** [review]
Every time modifications are made the PR gets locked and require an admin review to be merged.

File: CITATION.cff (hunk 1 above)
Comment (silvavn) [reply CITATION.cff:36]: @arunkannawadi I am not completely sure, Zenodo just informs that the zip file will be collected and the DOI will be generated. I probably should get an email notification. If that's the case I can replace the dummy value when I get it.

**Comment by arunkannawadi** [comment]
Hmm, I'm quite sure that that wasn't the case before and that I was able to make changes. It's fine for now, but may lead to inconvenience in the future when we want to fix errors in a PR.

**Pull request merged**